### DISCUSSION TITLE
Is it possible to use XTTS v2 Commercially?

### DISCUSSION THREAD

**Discussion by hypervoxel** [idiap/coqui-ai-TTS#216] [Q&A] [answered]
Since Coqui is shutting down, and this is a fork, is it possible to use XTTS commercially?

**Comment by eginhard** [comment, marked as answer]
The license of the code ([MPL 2.0](https://github.com/idiap/coqui-ai-TTS/blob/dev/LICENSE.txt)) and the [models](https://github.com/idiap/coqui-ai-TTS/blob/dev/TTS/.models.json) remain of course unchanged (we are not the license holders), so you can continue to use everything under the previous terms. For XTTS, this is the [CPML](https://coqui.ai/cpml), which does not allow commercial use.

**Reply by Lgomes-GvizStudio** [reply]
is there any way to contact the the guys behind the xtts v2, i am building a chatbot using the model , i want to use commercially .

**Reply by eginhard** [reply]
No, the company shut down in 2023, so there is no longer any way to purchase a commercial license.

**Reply by BorasyKong** [reply]
This is a grey area. Technically, you can because essential who's gonna sue you. Unless of course the license gets transferred to another company somehow.

**Comment by hypervoxel** [comment]
Thank you. I think I understand why they are shutting down. Surely they would want people to use this Commercially?
